Detectors/TPC/workflow/src/CATrackerSpec.cxx

```diff
@@ -74,7 +82,104 @@ DataProcessorSpec getCATrackerSpec(bool processMC, std::vector<int> const& input
       auto& tracker = processAttributes->tracker;
       parser = std::make_unique<ClusterGroupParser>();
       tracker = std::make_unique<o2::tpc::TPCCATracking>();
-      if (tracker->initialize(options.c_str()) != 0) {
+
+      // Prepare initialization of CATracker - we parse the deprecated option string here,
+      // and create the proper configuration objects for compatibility.
+      // This should go away eventually.
+
+      // Default Settings
+      float solenoidBz = -5.00668; // B-field
+      float refX = 1000.;          // transport tracks to this x after tracking, >500 for disabling
```

Comment:
@davidrohr: just noticed: is there a reason not use by default refX=83, expected in the matching?

Reply:
no particular reason, i can change that.

Reply:
done